build.gradle

```diff
@@ -55,6 +59,20 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
     }
 }
 
+dependencies {
+    compile "com.amazon.ion:ion-java:$ionVersion"
+
+    // using kotlin to make tests less verbose
+    testCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
+
+    // JUnit 5
+    testCompile "org.junit.jupiter:junit-jupiter-api:$junitVersion"
```

Comment:
Let's pick a single convention for variables:  $kotlin_version/$junit_version/$ion_version, or $kotlinVersion/$junitVersion/$ionVersion.